src/mlpack/tests/random_forest_test.cpp

```diff
@@ -558,3 +558,54 @@ TEST_CASE("WarmStartTreesPredictionsQualityTest", "[RandomForestTest]")
 
   REQUIRE(newCorrect - oldCorrect >= 0);
 }
+
+/**
+ * Ensure that the Extra Trees algorithm gives decent accuracy.
+ */
+TEST_CASE("ExtraTreesAccuracyTest", "[RandomForestTest]")
+{
+  // Load the vc2 dataset.
```

Comment:
Looks like this is not the `vc2` dataset.

Reply:
Thanks. Nice catch!